docs/content/advanced/backend/subscribers/create-subscriber.md

```diff
@@ -0,0 +1,92 @@
+
+Natively in Medusa there are subscribers to handle different events. However, you can also create your own custom subscribers. 
+
+Custom subscribers reside in `src/subscribers` within a JavaScript file. Each file holds a class, which is the subscriber. By convention, the class name should end with `Subscriber` and the file name should be the camel-case version of the class name without `Subscriber`. For example, the `WelcomeSubscriber` class is in the file `src/subscribers/welcome.js`.
+
+Whenever an event is emitted, the subscriber’s registered handler method is executed. The handler method receives as a parameter an object that holds data related to the event. For example, if an order is placed the `order.placed` event will be emitted and all the handlers will receive the order id in the parameter object.
+
+## Prerequisites
+
+Before you can use subscribers, whether your custom subscribers or Medusa’s subscribers, you need to have [Redis](https://redis.io) installed and setup on your machine.
+
+Then, you need to set your Redis URL in your Medusa server. By default, the Redis URL is `redis://localhost:6379`. If you use a different one, set the following environment variable in `.env`:
+
+```bash
+REDIS_URL=<YOUR_REDIS_URL>
+```
+
+After that, in `medusa-config.js`, you’ll need to comment out the following line:
+
+```jsx
+module.exports = {
+  projectConfig: {
+    redis_url: REDIS_URL, //this line is commented out
+    ...
+  }
+}
+```
+
+After that, you should be able to listen to events on your server.
+
+## Implementation
+
+To create a subscriber, you’ll need to create a file in `src/subscribers` with the camel-case version of your subscriber’s name without the word `Subscriber`.
+
+Then, in the constructor of your subscriber, you should listen to events using `eventBusService.subscribe` , where `eventBusService` is a service injected into your subscriber’s constructor. 
+
+The `eventBusService.subscribe` method receives the name of the event as a first parameter and as a second parameter a method in your subscriber that will handle this event.
+
+For example, here is the `OrderNotifierSubscriber` class which is created in `src/subscribers/orderNotifier.js`:
+
+```jsx
+class OrderNotifierSubscriber {
+  constructor({ eventBusService }) {
+    eventBusService.subscribe("order.placed", this.handleOrder);
+  }
+
+  handleOrder = async (data) => {
+    console.log("New Order: " + data.id)
+  };
+}
+
+export default OrderNotifierSubscriber;
+```
+
+This subscriber will register the method `handleOrder` as one of the handlers of the `order.placed` event. The method `handleOrder` will be executed every time an order is placed, and it will receive the order ID in the `data` parameter. You can then use the order’s details to perform any kind of task you need.
```

Comment:
We should maybe be clear on the fact that `data` only contains the `id` of the resource that was processed. Not the entire data so:

`"order.placed", { "id": "order_...." }`

Reply:
I added a note below about it